docs/source/concept_models/vivarium_iv_iron/concept_model.rst

```diff
@@ -129,16 +129,58 @@ Iron supplementation during pregnancy can help meet the increased iron demands d
 4.2 Demographics
 ----------------
 
+4.2.1 Locations
+~~~~~~~~~~~~~~~
+
+Locations of interest to this project: 
+
+- Sub-Saharn Africa (location_type=superregion; location_id=166)
+- South Asia (location_type=region; location_id=159)
+- All low and middle income countries
```

Comment:
Is there a location ID for LMICs?

Reply:
Yes, there are several different classifications and I haven't figured out exactly which one(s) will be best for us to use. I am thinking 44577 and 44578, but want to check a few more things to confirm. I added a to-do note for now and will follow-up.
![image](https://user-images.githubusercontent.com/54918472/146104412-e05b3f2f-d2a2-4371-9038-30c832a97965.png)